debian/testbed/nextcloud/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM debian
+RUN apt-get update
+#just to speedup after changing the deb
+RUN apt-get install -y php7.0-xml php7.0 php7.0-cgi php7.0-cli php7.0-gd php7.0-curl php7.0-zip php7.0-mysql php7.0-mbstring
```

Comment:
Join these lines, see [guide](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#apt-get)

Reply:
i wil delete the line.